src/ast/grouping.rs

```diff
@@ -0,0 +1,76 @@
+use crate::ast::{Column, DatabaseValue};
+
+pub type GroupByDefinition = (DatabaseValue);
```

Comment:
Why not just using `DatabaseValue` in `Grouping`? What does this type definition add to the code?

Reply:
Ah, now I get it. Limitation what can be used in the grouping!